storage/src/adb/any/variable/mod.rs

```diff
@@ -597,21 +605,21 @@ impl<E: RStorage + Clock + Metrics, K: Array, V: Codec, H: CHasher, T: Translato
     pub async fn historical_proof(
         &self,
         op_count: u64,
-        start_loc: u64,
+        start_loc: Location,
         max_ops: u64,
     ) -> Result<(Proof<H::Digest>, Vec<Operation<K, V>>), Error> {
         assert!(op_count <= self.op_count());
         assert!(start_loc < op_count);
 
-        let end_loc = std::cmp::min(op_count, start_loc + max_ops);
-        let mmr_size = leaf_loc_to_pos(op_count);
+        let end_loc = std::cmp::min(op_count, start_loc.as_u64() + max_ops);
```

Comment:
mind changing this addition to checked_add?

Reply:
Done